### PR TITLE
DOC: Update absolute_beginners.rst

### PR DESCRIPTION
**Pull request by sebasmos**
Updating example descriptions

### CODE REVIEW

File: doc/source/user/absolute_beginners.rst [hunk 1]
```diff
@@ -1575,6 +1575,8 @@ easiest way to do this is to use
 
 .. image:: images/np_pandas.png
 
+The read_csv function from the Pandas library is used to read a CSV file. The header=0 argument is passed to the function to indicate that the first row of the CSV file should be used as column names for the resulting DataFrame. The .values method is then called on the DataFrame to convert it to a NumPy array.
```
Comment (charris) [inline doc/source/user/absolute_beginners.rst:1578]: Be good to shorten these long lines, you need had line breaks.

**Review by sebasmos — COMMENTED** [review]
Thank Charles! Done, please take a look

**Review by sebasmos — COMMENTED** [review]
Broke into smaller lines

File: doc/source/user/absolute_beginners.rst [hunk 2]
```diff
@@ -1575,9 +1577,18 @@ easiest way to do this is to use
...
 .. image:: images/np_pandas.png
 
-It's simple to use Pandas in order to export your array as well. If you are new
-to NumPy, you may want to  create a Pandas dataframe from the values in your
-array and then write the data frame to a CSV file with Pandas.
+The read_csv function from the Pandas library is used to read a
+CSV file. The header=0 argument is passed to the function to indicate
+that the first row of the CSV file should be used as column names
+for the resulting DataFrame. The .values method is then called on the 
+DataFrame to convert it to a NumPy array.
```
Comment (rossbar) [inline doc/source/user/absolute_beginners.rst:1584]: I'm not sure that adding narrative text explaining the Pandas API is really an improvement to the numpy tutorial. Furthermore I don't think the recommendations are necessarily correct - for example I believe `to_numpy` is preferred to `values`. Therefore I'm -1 on adding this paragraph.

**Comment by Mukulikaa** [comment]
Closing, as changes are not desirable in the context of the NumPy documentation. Reopen if someone feels otherwise.

**Pull request closed without merging**